RootOutputer.h

```diff
@@ -26,6 +26,7 @@ class RootOutputer :public OutputerBase {
     int basketSize_=16384;
     int treeMaxVirtualSize_=-1;
     int autoFlush_=-1;
+    int cacheSize_=102400;
```

Comment:
Since this is a new feature, I'd prefer to keep the behavior the same as before so
```suggestion
    int cacheSize_=0;
```